pagopa/io-app
# feat: [IOBP-226] Addition of biometric security in  idpay code flow

## Short description
THIS PR DEPENDS ON https://github.com/pagopa/io-app/pull/4976
addition of biometric check state to the idpay machine

https://github.com/pagopa/io-app/assets/60693085/3f75bf13-02e8-495c-8717-77f686ca4862


## List of changes proposed in this pull request
- addition of full flow entry in idpay code testing page
- addition of biometric check in idpay machine, with required accomodations

## How to test
with idpay's test flag active, navigate to profile>idpay code test > full flow and make sure the flow correctly navigates to "ONBOARDING_FAILURE" in cancel case and to "ONBOARDING_SUCCESS" in case of correct pin.

Right now the machine is "dumb", so we are using mocks to make it behave as if no code was to be generated -- aka a code is already onboarded in the app
